shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/TCLStatement.g4

```diff
@@ -50,4 +50,16 @@ rollback
 
 savepoint
     : SAVEPOINT
-    ;
\ No newline at end of file
+    ;
+
+transactionCharacteristic_
```

Comment:
If you need to visit `transactionCharacteristic_ `, please rename it to `transactionCharacteristic`.
It is also necessary to check other rule names.